### DISCUSSION TITLE
Throwing an exception when starting the main branch version project.

### DISCUSSION THREAD

**Discussion by qian-o** [HexaEngine/HexaEngine#77] [Q&A] [answered]
![image](https://github.com/HexaEngine/HexaEngine/assets/84434846/48ff7f9a-7c9a-4c25-a658-6b1747e33dc8)

**Comment by JunaMeinhold** [comment]
I committed a fix to the main branch, could you please check again and if it doesn't work please provide some information about your system specs like os, os-version, monitor configuration and gpu for further analysis.

**Reply by qian-o** [reply]
Okay, I will try tomorrow. If there are any other issues, I will upload the device information.

**Reply by qian-o** [reply]
It's still not working. I'm using a laptop with an external monitor, and I have closed the laptop screen.

![image](https://github.com/HexaEngine/HexaEngine/assets/84434846/7e1073bc-4c84-4969-8393-1f772086faec)

[CPU] AMD Ryzen 7 5800H with Radeon Graphics
[GPU 0] NVIDIA GeForce RTX 3050 Laptop GPU
[GPU 1] AMD Radeon(TM) Graphics

**Comment by JunaMeinhold** [comment, marked as answer]
I have fixed the bug. It was an issue with GPU selection (iGPU + dedicated GPU) where the backend was selecting the iGPU and not the dedicated GPU.

PS: You should select Editor as the startup project, the app project is the stripped down version for game publishing and is not really used except for debugging. Since the editor automatically generates this target while publishing.

**Reply by qian-o** [reply]
Great, it ran successfully, but some characters cannot be displayed correctly due to regional issues.
![image](https://github.com/HexaEngine/HexaEngine/assets/84434846/74fe918a-5224-4b8a-aadc-f86efe196100)

**Reply by JunaMeinhold** [reply]
Just restart the editor. This happens sometimes and I don't know why, still investigating.
